src/NodaTime/Period.cs

```diff
@@ -725,33 +726,30 @@ public Duration ToDuration()
         [NotNull]
         public PeriodBuilder ToBuilder() => new PeriodBuilder(this);
 
-        // FIXME(2.0): Normalize to a particular fraction-of-second type?
         /// <summary>
         /// Returns a normalized version of this period, such that equivalent (but potentially non-equal) periods are
         /// changed to the same representation.
         /// </summary>
         /// <remarks>
         /// Months and years are unchanged
         /// (as they can vary in length), but weeks are multiplied by 7 and added to the
-        /// Days property, and all time properties are normalized to their natural range
-        /// (where ticks are "within a millisecond"), adding to the larger property where
-        /// necessary. So for example, a period of 25 hours becomes a period of 1 day
+        /// Days property, and all time properties are normalized to their natural range.
+        /// Subsecond values are normalized to millisecond and "nanosecond within millisecond" values.
+        /// So for example, a period of 25 hours becomes a period of 1 day
```

Comment:
Could do - although the fact that we don't support microseconds as a unit makes it a little odd. (In order to test such an assertion, you'd probably create a period of 1500750000 nanoseconds.)